test/core/Leaf.t.sol

```diff
@@ -286,12 +286,18 @@ contract TickTest is Test {
     Field level1 = Field.wrap(bound(_level1, 1, uint(LEVEL1_SIZE) - 1));
     Field level2 = Field.wrap(bound(_level2, 1, uint(LEVEL2_SIZE) - 1));
     Field level3 = Field.wrap(bound(_level3, 1, uint(LEVEL3_SIZE) - 1));
-    Tick tick = TickLib.tickFromBranch(tickPosInLeaf, level0, level1, level2, level3);
+    MgvStructs.LocalPacked local;
+    local = local.tickPosInLeaf(tickPosInLeaf);
+    local = local.level0(level0);
+    local = local.level1(level1);
+    local = local.level2(level2);
+    local = local.level3(level3);
```

Comment:
This is the simplest case where only one bit is set in each level. Would be good to also test this where some higher bits are also set.

Reply:
Doesn't this allow any number of bits to be set except 0, up to the size of the level?
```
    Field level0 = Field.wrap(bound(_level0, 1, uint(LEVEL0_SIZE) - 1));
    Field level1 = Field.wrap(bound(_level1, 1, uint(LEVEL1_SIZE) - 1));
    Field level2 = Field.wrap(bound(_level2, 1, uint(LEVEL2_SIZE) - 1));
    Field level3 = Field.wrap(bound(_level3, 1, uint(LEVEL3_SIZE) - 1));
 ```

Reply:
doh, you're right, my bad - sorry!